lms/templates/navigation.html

```diff
@@ -51,7 +52,13 @@
     <h1 class="logo">
       <a href="${marketing_link('ROOT')}">
         <%block name="navigation_logo">
-            <img src="${branding_api.get_logo_url(is_secure)}" alt="${_("{platform_name} Home Page").format(platform_name=static.get_platform_name())}"/>
+        <%
+        logo_url = None
```

Comment:
Do we need to set this `logo_url` value here?  `get_enterprise_customer_logo_url` should return None at the very least, right?